src/plays/web3-staking-dapp/Web3StakingDapp.js

```diff
@@ -0,0 +1,273 @@
+import PlayHeader from 'common/playlists/PlayHeader';
+import './styles.css';
+import React, { useState, useEffect } from "react";
+import artifact from "./artifacts/contracts/Staking.sol/Staking.json";
+import { ethers } from "ethers";
+import Navbar from "./components/Navbar";
+import maticLogo from "./images/polygon-matic-logo.svg";
+import { FaRupeeSign } from 'react-icons/fa';
+
+// WARNING: Do not change the entry componenet name
+function Web3StakingDapp(props) {
+
+  // Your Code Start below.
+
+  const ContractAddress = '0x12163B070B97f06F5061D93164D960bbFCfdf965';
+
```

Comment:
Move this to constant.

Reply:
So I have to create a file called constant,js and then I have to keep the contract address there?
This is what you are saying

Reply:
Yes, Exactly.

Reply:
ok